diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter2/kdoc/CommentsFormatting.kt

```diff
@@ -56,8 +56,6 @@ class CommentsFormatting(configRules: List<RulesConfig>) : DiktatRule(
         IF_ELSE_COMMENTS, WRONG_NEWLINES_AROUND_KDOC)) {
     /**
      * @param node
-     * @param autoCorrect
-     * @param emit
```

Comment:
Hmm, why hasn't it been spotted by diktat? IIRC we have a check for outdated parameters in KDocs. Could you have a look and open a new issue if there is really a bug?

Reply:
[open](https://github.com/cqfn/diKTat/issues/859)

Reply:
We already have this logic, actually you yourself have implemented it in #237. So this issue should be instead to check why diktat didn't spot wrong params in this particular case.

Reply:
Okey, changed issue's description